### DISCUSSION TITLE
Integration for Windows 10/11 media interface?

### DISCUSSION THREAD

**Discussion by akippnn** [ppy/osu#17417] [Ideas] [answered]
This wrapper explains what I mean:
https://github.com/DubyaDude/WindowsMediaController

A neat nice-to-have. Allows for convenient control playback as well as media information.

*Although, at the moment, this is not very useful with Windows 11 because the media interface has been tucked into Quick Settings.*

**Comment by bdach** [comment, marked as answer]
Not happening. osu! is not a music player.
